console/console-init/ui/src/modules/address-space/containers/MessagingToolbarContainer/MessagingToolbarContainer.tsx

```diff
@@ -97,16 +97,23 @@ export const MessagingToolbarContainer: React.FunctionComponent<IMessagingToolba
     setNameInput(undefined);
   };
 
-  const onChangeNameInput = async (value: string) => {
+  const getAddressSpaceForNameORNameSpace = async (
+    value: string,
+    propertyName: string
+  ) => {
     const response = await client.query<
       ISearchNameOrNameSpaceAddressSpaceListResponse
     >({
       query: RETURN_ALL_ADDRESS_SPACES_FOR_NAME_OR_NAMESPACE(
-        true,
+        propertyName,
         value.trim()
       ),
       fetchPolicy: FetchPolicy.NETWORK_ONLY
     });
+    return response;
+  };
+
+  const getOptions = (response: any, propertyname: string) => {
```

Comment:
Why is 'any' chosen as the response type?

Reply:
done